scrapy/utils/conf.py

```diff
@@ -72,6 +74,7 @@ def closest_scrapy_cfg(path='.', prevpath=None):
     path = os.path.abspath(path)
     cfgfile = os.path.join(path, 'scrapy.cfg')
     if os.path.exists(cfgfile):
+        logger.debug('Scrapy config file used: {}'.format(cfgfile))
         return cfgfile
     return closest_scrapy_cfg(os.path.dirname(path), path)
```

Comment:
Given that the function is recursive, isn't it the base case that will eventually be hit?